libs/server/Resp/AdminCommands.cs

```diff
@@ -542,13 +543,77 @@ private bool ProcessAdminCommands<TGarnetApi>(RespCommand command, ReadOnlySpan<
             {
                 return ProcessACLCommands(bufSpan, count);
             }
-            else if ((command == RespCommand.REGISTERCS))
+            else if (command == RespCommand.REGISTERCS)
             {
                 return NetworkREGISTERCS(count, recvBufferPtr + readHead, storeWrapper.customCommandManager);
             }
+            else if (command == RespCommand.ASYNC)
+            {
+                if (respProtocolVersion > 2 && count == 1)
```

Comment:
Is it planned to support the OPTIN, YES, NO options?

Reply:
Not in this PR - it does not seem to be necessary for the intended use case.